webui/react/src/hooks/useFeature.ts

```diff
@@ -0,0 +1,25 @@
+import queryString from 'query-string';
+import { useLocation } from 'react-router-dom';
+
+import { useStore } from 'contexts/Store';
+
+interface FeatureHook {
+  isOnRbac: () => boolean
+}
+
+interface Queries {
+  rbac?: string;
+}
+
+const useFeature = (): FeatureHook => {
+  return { isOnRbac: () => IsOnRbac() };
+};
+
+const IsOnRbac = (): boolean => {
+  const { info: { rbacEnabled } } = useStore();
+  const location = useLocation();
+  const { rbac }: Queries = queryString.parse(location.search);
```

Comment:
We should only check the url on page load, not whenever it changes. You can get the url directly from the JS standard API. Also can you check for `f_rbac` instead of just `rbac`.